test/test-helpers.js

```diff
@@ -372,31 +373,29 @@ export const standardXHRResponse = function(request, data) {
     manifestName = request.url;
   }
 
+  const isContainerRequest = request.mimeTypeOverride === 'text/plain; charset=x-user-defined';
```

Comment:
Since this could be for partial data downloads in any case, we may want to rename to something like `isPartialRequest`